react-native/react/native/notification-listeners.desktop.js

```diff
@@ -23,11 +27,43 @@ export default {
       [enums.kbfs.FSStatusCode.error]: 'Errored'
     }[notification.statusCode]
 
-    const pubPriv = notification.publicTopLevelFolder ? '[Public]' : '[Private]'
+    const basedir = notification.filename.split('/')[0]
```

Comment:
better to use node helpers right?

https://nodejs.org/api/path.html#path_path_parse_pathstring